modules/host/storageobligations.go

```diff
@@ -764,7 +764,7 @@ func (h *Host) handleActionItem(so *storageObligation) {
 
 		// Queue another action item to check whether there the storage proof
 		// got confirmed.
-		err = h.queueActionItem(h.blockHeight+types.BlockHeight(storageProofConfirmations), so.id())
+		err = h.queueActionItem(so.proofDeadline(), so.id())
```

Comment:
makes me uneasy changing host stuff so close to release, but this seems simple enough that I trust it doesn't need live testing